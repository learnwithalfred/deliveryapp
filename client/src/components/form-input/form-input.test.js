import React from "react";
import { FormInput } from "./form-input.component";
import { shallow } from "enzyme";

describe("Form Input should be rendered", () => {
  let wrapper;
  let handleChange = jest.fn();
  beforeEach(() => {
    const mockProps = {
      mockhandleChange: handleChange,
      mocklabel: "img",
      otherProps: "",
    };
    wrapper = shallow(<FormInput {...mockProps} />);
  });

  it("Should snapshot Form Input", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
