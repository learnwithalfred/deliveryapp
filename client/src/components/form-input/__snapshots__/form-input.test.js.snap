// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Input should be rendered Should snapshot Form Input 1`] = `ShallowWrapper {}`;
